addons/web/static/tests/mobile/views/calendar_view_tests.js

```diff
@@ -360,6 +360,10 @@ QUnit.module("Views", ({ beforeEach }) => {
         await tap(target, ".fc-day-top[data-date='2016-02-05']");
         await nextTick(); // switch renderer
         await nextTick(); // await breadcrumb update
+        assert.strictEqual(
+            document.querySelector(".o_calendar_container .o_calendar_header h5").textContent,
+            "5 February 2016"
+        );
```

Comment:
Coucou @aab-odoo 

I added that assert to do the same assert than we did below before the changes made in that PR. 🙂